include/modules/meta/base.inl

```diff
@@ -250,9 +250,16 @@ namespace modules {
         }
       }
 
+      string tag_content;
+      bool tag_built{false};
       string tag = value.substr(start, end - start + 1);
-      bool tag_built = CONST_MOD(Impl).build(&tag_builder, tag);
-      string tag_content = tag_builder.flush();
+      if (format->tags.find(tag) != format->tags.end()) {
+        tag_built = CONST_MOD(Impl).build(&tag_builder, tag);
+        tag_content = tag_builder.flush();
+      } else {
+        tag_built = true;
+        tag_content = tag;
```

Comment:
In this case, it should probably set `tag_built = false` and add the tag name to `non_tag`.

Because right now, an invalid tag also applies `format->spacing`.
For example if you have

```dosini
format = <label> <invalid>
format-spacing = 3
label = Foo
```

This will be rendered as `Foo    <invalid>` instead of `Foo <invalid>`